tests/recipe_training_tests/shortened_recipes_accuracy_test.py

```diff
@@ -3,9 +3,10 @@
 import os
 import torch
 
-from super_gradients.common.environment.checkpoints_dir_utils import get_checkpoints_dir_path
+from super_gradients.common.environment.checkpoints_dir_utils import get_latest_run_dir_path
```

Comment:
I cannot find get_latest_run_dir_path in master. This feature branch also don't have this function added. Am I missing something @ofrimasad ?

Reply:
true.
moving this to draft mode until @Louis-Dupont fix the DDP <> run Id issue